pkg/virt-handler/node-labeller/node_labeller.go

```diff
@@ -138,6 +134,41 @@ func (n *NodeLabeller) execute() bool {
 	return true
 }
 
+func (n *NodeLabeller) loadAll() error {
+	if !n.loadedCPUInfo {
+		err := n.loadCPUInfo()
+		if err != nil {
+			n.logger.Errorf("node-labeller could not load cpu info: " + err.Error())
+			return err
+		}
+		n.loadedCPUInfo = true
+	}
+
+	if !n.loadedSupportedFeatures {
+		err := n.loadHostSupportedFeatures()
+		if err != nil {
+			n.logger.Errorf("node-labeller could not load supported features: " + err.Error())
+			return err
+		}
+		n.loadedSupportedFeatures = true
```

Comment:
What you think about updating `loadedSupportedFeatures` boolean as part of  `loadHostSupportedFeatures` function?